File: Assets/Editor/GMCommand.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEditor;
using System;
using XLua;

[CSharpCallLua]
public class GMCommand
{
    [MenuItem("GMCommand/清除已拥有英雄数据")]
    public static void ClearOwnHeroData()
    {
        ILuaGM luaFunc = LuaManager.Instance.MyLuaEnv.Global.Get<ILuaGM>("GMCommand");
        luaFunc.clearOwnHeroInfo();
    }

    [MenuItem("GMCommand/清除已上阵英雄数据")]
    public static void ClearDeployHeroData()
    {
        ILuaGM luaFunc = LuaManager.Instance.MyLuaEnv.Global.Get<ILuaGM>("GMCommand");
        luaFunc.clearDeployHeroInfo();
    }
}

File: Assets/LuaScripts/4_UI/GMCommand.lua
GMCommand = BaseClass("GMCommand")
-- 初始化函数
function GMCommand:__init()

end

function GMCommand.clearOwnHeroInfo()
    LocalModel:Instance():ClearOwnHero()
    LocalModel:Instance():ClearDeployHero()
end

function GMCommand.clearDeployHeroInfo()
    LocalModel:Instance():ClearDeployHero()
end

return GMCommand

